app/_data/docs_nav_kuma_dev.yml

```diff
@@ -268,6 +268,8 @@ items:
         items:
           - text: Disabling IPv6
             url: "/networking/ipv6/#disabling-ipv6"
+      - text: Non-mesh traffic
+        url: /networking/non-mesh-traffic/
```

Comment:
Doesn't that goes under `transparent-proxy` ?

Reply:
"non-mesh-traffic.md" is a separate page that contains all the information about the outside mesh traffic. So it's not under "transparent-proxy"